multimedia/gst1-plugins-base/patches/004-Synchronize-playback-support-using-alsa-start-at.patch

```diff
@@ -0,0 +1,3029 @@
+From 8c4bef5f62b260206d2bcd7cb9ff0ba29eaec602 Mon Sep 17 00:00:00 2001
+From: Abhilash Tuse <Abhilash.Tuse@imgtec.com>
+Date: Fri, 25 Nov 2016 18:23:19 +0530
+Subject: [PATCH] Synchronize playback support using alsa start-at
+
+Add gstalsaclock library that implements a custom GstClock that uses
+pcm driver interface to read high-precision audio hardware clock
+timestamps.
+
+Add property 'start-at' in alsasink for additional delay
+of the sink in nanoseconds to sync playback start in hardware.
+
+Change-Id: I1189aaa8e2d66f664afe89a63490b792ca928143
+Signed-off-by: Abhimanyu Vishwakarma <Abhimanyu.Vishwakarma@imgtec.com>
+Signed-off-by: Matin Momin <Matin.Momin@imgtec.com>
+Signed-off-by: Sagar Tikore <Sagar.Tikore@imgtec.com>
+Signed-off-by: Marcin Nowakowski <marcin.nowakowski@imgtec.com>
```

Comment:
AFAIR there was no work done by me here except for the gettext version update, which must not be part of this patch.